ng-dev/pr/merge/target-label.ts

```diff
@@ -30,6 +31,12 @@ export function getTargetLabelFromPullRequest(
   config: Pick<MergeConfig, 'labels'>,
   labels: string[],
 ): TargetLabel {
+  if (config.labels === false) {
```

Comment:
Nit: I think it would be cleaner to create the labels config when this method is invoked instead. This method shouldn't really fetch the ng-dev config or deal with the defaults IMO.

Reply:
Also, the git client with the main branch name would exist there already

Reply:
Would it make sense instead to change the config to drop `labels` and add `noBranchTargeting?: boolean`

Then branch targeting can turned off entirely, so the options become either use the common branch targeting or just perform merges to wherever the PR targets in Github?

Reply:
great idea 👍